src/conversion/modelconversiondispatcher.js

```diff
@@ -188,7 +198,9 @@ export default class ModelConversionDispatcher {
 				range: itemRange
 			};
 
-			this._testAndFire( 'insert', data, consumable );
+			const name = item.name || '$text';
+
+			this._testAndFire( 'insert', consumable, data, name );
```

Comment:
In fact the whole idea of the `name` parameter, when you have `item` is strange for me.